src/aktualizr_info/main.cc

```diff
@@ -142,6 +143,22 @@ int main(int argc, char **argv) {
       std::cout << "Private key:" << std::endl << priv << std::endl;
     }
 
+    auto pacman = PackageManagerFactory::makePackageManager(config.pacman, storage, nullptr);
+
+    Uptane::Target current_target = pacman->getCurrent();
+
+    if (current_target.IsValid()) {
+      std::cout << "Current primary ecu running version: " << current_target.sha256Hash() << std::endl;
+    }
```

Comment:
Should we print something if the target is not valid?

Reply:
Good suggestion!